docs/users/plugins.md

```diff
@@ -41,6 +41,11 @@ assert formatted == '```python\n"""black converts quotes"""\n```\n'
     <td><code>python</code></td>
     <td></td>
   </tr>
+  <tr>
+    <td><a href="https://github.com/Freed-Wu/mdformat-ruff">mdformat-ruff</a></td>
```

Comment:
Can you please move this below `mdformat-gofmt` so that it is sorted alphabetically?